tools/pi_db_tools/delta_pi_calc.xml

```diff
@@ -1,8 +1,11 @@
 <tool id="calc_delta_pi" name="Add delta pI" version="1.1">
+    <description>to peptide table</description>
     <requirements>
         <requirement type="package" version="3.6">python</requirement>
     </requirements>
-    <description>to peptide table</description>
+    <stdio>
```

Comment:
Can you please use https://docs.galaxyproject.org/en/latest/dev/schema.html#id5 instead of stdio. `exit_code` should be the one which you need.

Reply:
Thanks, I had no idea this could be spec'ed on the command element!